packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__mocks__/testUtils.ts

```diff
@@ -130,12 +138,101 @@ export const mockFace: Face = {
   leftEar: [200, 200],
 };
 
-export const mockOvalDetails: LivenessOvalDetails = {
+// Values generated by passing mockFace into generateBboxFromLandmarks and printing intermediate values
+export const mockFaceHeight = 0; // 1.68 * [(ALPHA * pd + GAMMA * fh) / 2];
+export const mockFaceCenterY = 200; // eyeCenter[1];
+
+export const mockAboveFrameFace: Face = {
```

Comment:
Would like to move away from using this file with the intention of removing it entirely. Sharing test details between files makes for brittle testing. Prefer `mockAboveFrameFace` be declared in the test file that uses it. Same feedback for all additions to this file